ubuntu/debian/libgz-physics-dev.install

```diff
@@ -1 +1,7 @@
 usr/lib/*/cmake/*-physics[0-99]-all/*-physics[0-99]-all*
+
+usr/include/*/physics*/*/physics/dartsim*/*
```

Comment:
I'd rather not put dartsim specific stuff in `libgz-physics-dev`. If we need to keep the cmake files, I think it's best to keep `libgz-physics-dartsim-dev` and put them there.

Reply:
reverted in [c53b289](https://github.com/gazebo-release/gz-physics7-release/pull/3/commits/c53b28961f83fb997b31f1c5083f7e6c71442618), move DART to suggested in [679d4ae](https://github.com/gazebo-release/gz-physics7-release/pull/3/commits/679d4ae9673f2d74386d03fe6872d0ed62458703)